linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_maximum_age_login_defs/bash/shared.sh

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_ol,multi_platform_rhv,multi_platform_fedora,multi_platform_ubuntu
+# platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_ol,multi_platform_rhv,multi_platform_fedora,multi_platform_ubuntu,multi_platform_sle
```

Comment:
The rule is applicable everywhere, so technically, it makes sense to restrict the remediation. However, the remediation is generic and completely backed by the rule description, so I would rather suggest putting `multi_platform_all` here. WDYT @yuumasato or @ggbecker ?

Reply:
Fixed! Thanks :)